### PR TITLE
Data migration to merge India Book sources #259

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: geniza/footnotes/tests/test_footnotes_migrations.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+from django.db.migrations.executor import MigrationExecutor
+import pytest
+
+
+# migration test case adapted from
+# https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
+# and copied from mep-django
+
+
+@pytest.mark.last
```
Comment (thatbudakguy) [inline geniza/footnotes/tests/test_footnotes_migrations.py:14]: does the decorator do anything here? does this thing actually get run as a (no-op) test?

**Review by thatbudakguy — APPROVED** [review]
everything looks great here! migration logic is really easy to follow.

File: geniza/footnotes/tests/test_footnotes_migrations.py (hunk 1 above)
Comment (thatbudakguy) [reply geniza/footnotes/tests/test_footnotes_migrations.py:14]: relatedly: if we apply the decorator here do we also need it on tests that subclass this?

Comment (rlskoeser) [reply geniza/footnotes/tests/test_footnotes_migrations.py:14]: I remember testing before to see if I could just put it on the base classes and not the subclasses, and that didn't work; it needs to be on each specific test class. It probably doesn't do anything on this one (except possibly serve as a reminder that subclasses should be run last). Should I remove and add a comment?

Comment (thatbudakguy) [reply geniza/footnotes/tests/test_footnotes_migrations.py:14]: yeah, that sounds right. i think removing and adding a comment would be perfect

**Pull request merged**